example/src/ListWithHeader/Header.tsx

```diff
@@ -56,28 +57,46 @@ function HeaderNative(props: HeaderProps) {
     };
   });
 
+  const collapsedCoefficient = 0.7;
+  const openCoefficient = Platform.OS === 'macos' ? 1 : 0.5;
+  const padding = Platform.OS === 'macos' ? 10 : 0;
+  const horizontalOffset = Platform.OS === 'macos' ? 50 : 0; // todo: relative to width
```

Comment:
What about this `todo`? Will it be addressed in other PR?

Reply:
Same here